### PR TITLE
App/Activity: Use other/tensors in mobileNet filter

### PR DESCRIPTION
**Pull request by niley7464**
This patch changes tensor format from other/tensor to other/tensors.
other/tensor is obsoleted.

### CODE REVIEW

**Review by wooksong — COMMENTED** [review]
Q: Is ```other/tensors,num_tensors=1``` equivalent with ```other/tensor```?

**Comment by niley7464** [comment]
> Q: Is `other/tensors,num_tensors=1` equivalent with `other/tensor`?

Yes. There are some information about data types below.
https://nnstreamer.github.io/data-type-and-flow-control.html#updates-with-20

**Review by myungjoo — APPROVED** [review]

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt [hunk 1]
```diff
@@ -134,9 +134,9 @@ class MainActivity : ComponentActivity() {
 
         val mobileNet = File("$path/mobilenet_v1_1.0_224_quant.tflite")
         val mobileNetFilter =
-            "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
+            "other/tensors,num_tensors=1,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```
Comment (myungjoo) [inline ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt:137]: dimension ==> dimensions
type ==> types

**Review by myungjoo — CHANGES_REQUESTED** [review]
With other/tensors, they are plural .

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt (hunk 1 above)
Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt:137]: Oh I forgot to change dimension and type.
I will also update nnstreamer-example's Tizen web application too!
Thank you :)

Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt]: You already fixed Tizen web example 😂
Updated android part!

**Review by wooksong — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]

**Pull request merged**